packages/xdl/src/start/ExpoUpdatesManifestHandler.ts

```diff
@@ -0,0 +1,155 @@
+import { ExpoUpdatesManifest, getConfig } from '@expo/config';
+import { getRuntimeVersionForSDKVersion } from '@expo/sdk-runtime-versions';
+import express from 'express';
+import http from 'http';
+import { parse } from 'url';
+import uuid from 'uuid';
+
+import {
+  Analytics,
+  Config,
+  ProjectAssets,
+  ProjectUtils,
+  resolveEntryPoint,
+  UrlUtils,
+} from '../internal';
+import {
+  getBundleUrlAsync,
+  getExpoGoConfig,
+  getPackagerOptionsAsync,
+  stripPort,
+} from './ManifestHandler';
+
+function getPlatformFromRequest(req: express.Request | http.IncomingMessage): string {
+  const url = req.url ? parse(req.url, true) : null;
+  const platform = url?.query.platform || req.headers['expo-platform'];
+  if (!platform) {
+    throw new Error('Must specify expo-platform header or query parameter');
+  }
+  return String(platform);
+}
+
+export async function getManifestResponseAsync({
+  projectRoot,
+  platform,
+  host,
+}: {
+  projectRoot: string;
+  platform: string;
+  host?: string;
+}): Promise<{
+  body: ExpoUpdatesManifest;
+  headers: Map<string, number | string | readonly string[]>;
+}> {
+  const headers = new Map<string, any>();
+  // set required headers for expo-updates manifest specification
```

Comment:
technically Expo Updates is the protocol and expo-updates is the client library implementation
```suggestion
  // set required headers for Expo Updates manifest specification
```